python/ray/package.py

```diff
@@ -0,0 +1,90 @@
+import hashlib
+from pathlib import Path
+from zipfile import ZipFile
+
+from enum import Enum
+from ray.experimental import internal_kv
+
+PKG_DIR='/tmp'
+
+class Protocol(Enum):
+    GCS = "gcs://"
+    PIN_GCS = "pin_gcs://"
+    # These two are not supported
+    S3 = "s3://"
+    GITHUB = "github://"
+
+def _xor_bytes(left, right):
+    if left and right:
+        return bytes(a ^ b for (a, b) in zip(left, right))
+    return left or right
+
+def _zip_module(path, zip_handler):
+    for from_file_name in path.glob("**/*"):
+        # We include pyc for performance
+        if from_file_name.match("*.py") or from_file_name.match("*.pyc"):
+            to_file_name = from_file_name.relative_to(path.parent)
+            zip_handler.write(from_file_name, to_file_name)
```

Comment:
I think we should just include all files here, we can optimize later. Odds are if a datafile is here it's for a reason (used by the lib)